src/reducer.js

```diff
@@ -4,11 +4,13 @@ import filter from './reducers/filter';
 import lessons from './reducers/lesson';
 import isStudentMode from './reducers/mode';
 import language from './reducers/language';
+import teacherInfo from './reducers/teacherInfo';
 
 export default combineReducers({
   lessons,
   context,
   filter,
   isStudentMode,
-  language
+  language,
+  teacherInfo
```

Comment:
Jo, den bør vel kanskje det. Men jeg er ikke helt sikker på hvordan jeg bruker getTeacherInfo fra util.js uten å bruke en selector for å få bruke context? Er det det du tenker at jeg bør?